pkg/environment/domain/environment.go

```diff
@@ -54,3 +57,42 @@ func (e *Environment) SetDeleted() {
 	e.Environment.Deleted = true
 	e.Environment.UpdatedAt = time.Now().Unix()
 }
+
+type EnvironmentV2 struct {
+	*proto.EnvironmentV2
+}
+
+func NewEnvironmentV2(name, urlCode, description, projectID string, logger *zap.Logger) (*EnvironmentV2, error) {
+	id, err := uuid.NewUUID()
+	if err != nil {
+		logger.Error("failed to generate uuid", zap.Error(err))
+		return nil, err
+	}
+	uid := strings.ReplaceAll(id.String(), "-", "")
```

Comment:
Do we need to remove the hyphen from the id?

Reply:
@cre8ivejp 
We don't have to do it 👍 
I fixed!